apps/daimo-mobile/src/view/shared/SearchScreen.tsx

```diff
@@ -23,23 +30,47 @@ export function SearchScreen({
   textInputRef,
   mode,
 }: SearchScreenProps) {
+  const [inputFocused, setInputFocused] = useState(autoFocus ?? false);
   const contactsAccess = useContactsPermission();
 
   const placeHolderText = contactsAccess.permission?.granted
     ? "Search user, ENS, contact, or email..."
     : "Search user, ENS, email, or phone...";
 
+  const handleFocus = () => {
+    setInputFocused(true);
+  };
+
+  const handleBlur = () => {
+    setInputFocused(false);
+  };
+
+  const blurInput = () => {
+    textInputRef.current?.blur();
+  };
+
   return (
     <>
-      <View style={{ flexGrow: 0 }}>
-        <InputBig
-          innerRef={textInputRef}
-          autoFocus={autoFocus}
-          icon="search"
-          placeholder={placeHolderText}
-          value={prefix}
-          onChange={setPrefix}
-        />
+      <View style={{ flexDirection: "row", alignItems: "center", flexGrow: 0 }}>
+        <Animated.View layout={LinearTransition} style={{ flex: 1 }}>
+          <InputBig
+            innerRef={textInputRef}
+            autoFocus={autoFocus}
+            icon="search"
+            placeholder={placeHolderText}
+            value={prefix}
+            onChange={setPrefix}
+            onFocus={handleFocus}
+            onBlur={handleBlur}
+          />
+        </Animated.View>
+        {inputFocused && (
+          <Animated.View entering={FadeIn} exiting={FadeOut}>
+            <Pressable style={{ marginLeft: 16 }} onPress={blurInput}>
+              <TextBody>Cancel</TextBody>
```

Comment:
Could we make this text opacity / color change also in the pressed state? Makes it feel a bit more native

I did something similar [here](https://github.com/daimo-eth/daimo/blob/4eec1462de3781da50cc838a45928942fc49e1eb/apps/daimo-mobile/src/view/screen/InviteScreen.tsx#L118-L122)

Reply:
I made a new `PressableText` component that handles the pressed state, and I adapted the `InviteScreen` example you referenced to use it as well.